t/sql/mozart.sql

```diff
@@ -0,0 +1,188 @@
+-- Automatically generated, do not edit.
```

Comment:
Done in https://github.com/metabrainz/musicbrainz-server/commit/ef4d2597f23d78f8c39b4484b11af2c3cad47703